.jshintrc

```diff
@@ -4,7 +4,8 @@
     "require": false,
     "exports": true,
     "module": true,
-    "module.exports": true
+    "module.exports": true,
+    "JSON": true

```

Comment:
Hi @KunmyonChoi, so it appears that using the JSON global *should* have caused an error previously. We either started using it somewhere, or something changed with JSHint. I didn't have a chance to track this one down completely.